MIGRATING.md

```diff
@@ -28,6 +28,20 @@ The existing `.widgetrc` file used to configure the Widget has been **removed**.
 
 ## Migrating from 5.x to 6.x
 
+### Enable synthetic imports for TypeScript apps
+
+If you're using TypeScript, you'll need to enable synthetic imports in your `tsconfig.json`.
```

Comment:
Can we add instructions for normal Typescript applications? Angular is a special case that puts its TSConfig options inside a separate section.

Reply:
Fixed